Resources/public/js/views/dashboard/ez-dashboardblockallcontentview.js

```diff
@@ -0,0 +1,167 @@
+/*
+ * Copyright (C) eZ Systems AS. All rights reserved.
+ * For full copyright and license information view LICENSE file distributed with this source code.
+ */
+YUI.add('ez-dashboardblockallcontentview', function (Y) {
+    'use strict';
+
+    /**
+     * Provides the All Content Dashboard Block View class
+     *
+     * @module ez-dashboardblockallcontentview
+     */
+    Y.namespace('eZ');
+
+    var BLOCK_IDENTIFIER = 'all-content',
+        CLASS_LOADING = 'is-loading',
+        CLASS_ROW_SELECTED = 'is-row-selected',
+        SELECTOR_ROW = '.ez-allcontent-block-row',
+        SELECTOR_CONTENT = '.ez-allcontent-block-content',
+        EVENTS = {};
+
+    EVENTS[SELECTOR_ROW] = {tap: '_uiShowRowOptions'};
+
+    /**
+     * The all content dashboard block view
+     *
+     * @namespace eZ
+     * @class DashboardBlockAllContentView
+     * @constructor
+     * @extends eZ.DashboardBlockBaseView
+     */
+    Y.eZ.DashboardBlockAllContentView = Y.Base.create('dashboardBlockAllContentView', Y.eZ.DashboardBlockBaseView, [Y.eZ.AsynchronousView], {
+        initializer: function () {
+            this._fireMethod = this._fireSearchEvent;
+            /**
+             * Stores the click outside event handler
+             *
+             * @property _clickOutsideHandler
+             * @type {Object}
+             * @protected
+             */
+            this._clickOutsideHandler = null;
+            this.events = Y.merge(this.events, EVENTS);
+
+            this._set('identifier', BLOCK_IDENTIFIER);
+            this.get('container')
+                .addClass(this._generateViewClassName(Y.eZ.DashboardBlockBaseView.NAME))
+                .addClass(CLASS_LOADING);
+
+            this.after('itemsChange', this.render);
+            this.after('itemsChange', this._uiEndLoading);
+        },
+
+        /**
+         * Renders the dashboard block view
+         *
+         * @method render
+         * @return {eZ.DashboardBlockAllContentView} the view itself
+         */
+        render: function () {
+            var items = this.get('items').map(function (item) {
+                    return {
+                        content: item.content.toJSON(),
+                        contentType: item.contentType.toJSON(),
+                        location: item.location.toJSON()
+                    };
+                });
+
+            this.get('container').setHTML(this.template({items: items}));
```

Comment:
you should also pass the `loadingError` attribute value otherwise, the corresponding condition in the template is useless and a potential loading error is not handled.